packages/flutter_tools/test/general.shard/web/devfs_web_test.dart

```diff
@@ -633,6 +634,7 @@ void main() {
     expect(webDevFS.webAssetServer.getFile('main.dart'), isNotNull);
     expect(webDevFS.webAssetServer.getFile('manifest.json'), isNotNull);
     expect(webDevFS.webAssetServer.getFile('flutter_service_worker.js'), isNotNull);
+    expect(webDevFS.webAssetServer.getFile('version.json'), isNotNull);
```

Comment:
This isn't really ever testing that the contents are correct. Perhaps a unit test of your helper method to confirm it contains the correct details?

Reply:
Thanks for adding that test, this is fine now